packages/Takeover/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@paprika/takeover",
+  "version": "0.1.0",
+  "description": "Takeover component",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Takeover"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/button": "^0.1.13",
+    "@paprika/heading": "^0.1.6",
+    "@paprika/stylers": "^0.1.5",
+    "@paprika/tokens": "^0.1.4",
+    "focus-trap-react": "^6.0.0",
+    "prop-types": "^15.7.2",
+    "react-transition-group": "^4.3.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "styled-components": "^4.2.0"
```

Comment:
are we using L10n in this component if yes we should add it as a peerDependency if not ignore my comment and mark it as `resolve`.